client/components/product-icon/README.md

```diff
@@ -0,0 +1,59 @@
+ProductIcon Component
+=============
+
+ProductIcon component is a React component to display plans or products icons in SVG format.
+
+We have four different plans icons:
```

Comment:
Nit: It's generally more flexible if we just say that we have "the following" without specifying a number.

```suggestion
We have the following different plans icons:
```